io_stream.c

```diff
@@ -89,6 +120,15 @@ void* php_mongo_io_stream_connect(mongo_con_manager *manager, mongo_server_def *
 	} else {
 		mongo_manager_log(manager, MLOG_CON, MLOG_FINE, "Connecting to %s (%s) without connection timeout (default_socket_timeout will be used)", dsn, hash);
 	}
+#if PHP_VERSION_ID < 50600
+	/* Capture the server certificate, if SSL is enabled, so we can do hostname verification */
```

Comment:
Do you mean for this to *always* happen in pre-5.6, regardless of the context options?